docker/build_scripts/build.sh

```diff
@@ -72,6 +72,8 @@ yum -y erase wireless-tools gtk2 libX11 hicolor-icon-theme \
     avahi freetype bitstream-vera-fonts \
     ${PYTHON_COMPILE_DEPS}  > /dev/null 2>&1
 yum -y install ${MANYLINUX1_DEPS}
+# Add a soft-link from cmake28 to cmake
+ln -s `which cmake28` /usr/local/bin/cmake
```

Comment:
Do you think it is possible that someone wants the system cmake, and will get tripped up by this?  I've no idea myself - I haven't used the system one.